discovery.md

```diff
@@ -691,108 +695,102 @@ Content-Type: application/json
 
-For example, if a Discovery Endpoint has a Service called `MyService`, it is
-possible to rename that Service to `YourService` and then to add, or update,
-another Service to use `MyService` as its `name`. But this is only true if
-the rename of the first Service happens first the request message.
+If an `id` is not specified in any of the Services the entire request MUST
+fail.  If no Service can be found that corresponds to a specified `id`, the
+entire request MUST fail.  An `epoch` value MAY be omitted.  If a Service
+corresponding to a specified `id` has a higher recorded `epoch` value than the
+one given the entire request MUST fail.
 
-Likewise, while it might be less than optimal, it is technically possible for
-a request to update a Service multiple times if the same `id` is used more
-than once within a request.
+If other service attributes are included in the request, those SHOULD be
+ignored for the purposes of processing the request.
 
-The follow responses are defined by this specification:
-- `201 Created` if all the specified Services were processed successfully.
-  - If there was only one Service in the incoming request, then the response
-    MUST include an HTTP `Location` Header with a value that points to the
-    correspoding Service.
-  - If the request contained more than one Service, then the HTTP `Location`
-    Header MUST NOT appear in the response.
-  - The HTTP Response Body MUST include an array of `id` values whose order
-    and values MUST match the list of Services in the request.
+Service deletions MUST be processed in the order in which they are given. Thus,
+if a Service is specified multiple times the entire request MUST fail.
 
-Other responses are allowed, but not defined by this specification.
+The follow responses are defined by this specification:
+- `200 OK` if all the specified Services were deleted successfully.
+  - The HTTP Response Body MUST include an array of the Service values at the
+    time of deletion.  The order of those values MUST match that of the
+    Services in the request.
+- `400 Bad Request` if the first error encountered is a missing `id`
+- `404 Not Found` if the first error encountered is that there is no Service
+  with a given `id`
+- `409 Conflict` if the first error encountered is that a given epoch was less
+  than or equal to the existing epoch of the Service with a given `id`
 
 The format of the HTTP Request MUST adhere to the following:
 ```
 Content-Type: application/json
 
 [
   {
-    "url": "{url}",
-    "name": "{name}",
-    ... remainder of Service attributes ...
-  }
+    id: "{id}",      // REQUIRED
+    epoch: "{epoch}" // OPTIONAL
+  },
   ...
 ]
 ```
 
-The format of the HTTP Response MUST adhere to the following:
+The format of the `200 OK` HTTP Response MUST adhere to the following:
 ```
+200 OK
 Content-Type: application/json
 
 [
-  "id": "{id}"
+  {
+    "url": "{url}",
+    "name": "{name}",
+    ... remainder of Service attributes ...
```

Comment:
Are they expected to include the full service here? Should we just do IDs?  Just wondering, no strong opinion.

Reply:
I made [a comment](https://github.com/cloudevents/spec/pull/837/files#r664990150) meaning to call out this choice and ask if the two should be more aligned.